lib/pengine/native.c

```diff
@@ -652,7 +652,13 @@ native_output_string(pe_resource_t *rsc, const char *name, pe_node_t *node,
          * resource from becoming Master).
          */
         if (target_role_e == RSC_ROLE_STOPPED) {
-            have_flags = add_output_flag(outstr, "disabled", have_flags);
+            /* If the parent resource is disabled, don't add "(disabled)" to the
+             * output for this resource.  That will have been handled as part of
+             * the parent's output.
+             */
+            if (!rsc->parent || !pe__resource_is_disabled(rsc->parent)) {
+                have_flags = add_output_flag(outstr, "disabled", have_flags);
+            }
```

Comment:
I think we still want to show disabled by each member as well, in case someone is e.g. grepping for a member name

Reply:
Sure, that's easy enough to do.  Not having "disabled" printed out here was in the expected output of the bug, so that's what I went with.